### DISCUSSION TITLE
Any others encounter an issue where connecting to AWS ElastiCache continues to increase the number of connections over time

### DISCUSSION THREAD

**Discussion by jdgriffith** [taskforcesh/bullmq#2026] [General]
Our setup involves deploying our app into AWS Elastic Container Service (ECS) and our workers have a connection to an ElastiCache redis cluster. We have noticed the number of redis connections continue to climb over time in AWS. We have not been able to replicate this behavior locally. Has this happened to anyone else?

Any thoughts/ideas appreciated!

**Comment by manast** [comment]
I use AWS Elasticache and have not seen this behavior. Can you issue a CLIENT LIST command on your Redis instance to see what these connections look like, for example, workers do create connections with a special name that starts with the prefix "bull:". You will also be able to see the IP from which the connections are created and so on.

**Reply by jdgriffith** [reply]
Thanks for the response! I will see what I can find from issuing this command. That's the same command we have been using for local testing and I have noticed the "bull" prefix.

**Comment by jdgriffith** [comment]
Update: this was caused by another redis connection pool unrelated to BullMQ. We are still very happy with what you have built and it's been very solid for our team!
